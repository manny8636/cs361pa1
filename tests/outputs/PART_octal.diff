--- outputs/PART_octal.txt	2022-02-14 17:19:51.052433240 -0500
+++ expected/PART_octal.txt	2022-02-13 17:48:24.374294593 -0500
@@ -0,0 +1,2 @@
+INTEGER: 16
+Success!
