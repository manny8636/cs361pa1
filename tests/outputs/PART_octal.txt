<<<<<<< HEAD
Parsing ../data/octal.txt failed
=======
>>>>>>> 149606adc8b3f0cb79ed9f7c590375c37f62be46
