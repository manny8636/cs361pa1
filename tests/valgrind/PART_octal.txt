==149618== Memcheck, a memory error detector
==149618== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==149618== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==149618== Command: ../json -i ../data/octal.txt
==149618== 
==149618== 
==149618== Process terminating with default action of signal 2 (SIGINT)
==149618==    at 0x401EF56: __open_nocancel (open64_nocancel.c:45)
==149618==    by 0x4009165: open_verify.constprop.1 (dl-load.c:1536)
==149618==    by 0x400A5CF: _dl_map_object (dl-load.c:2174)
==149618==    by 0x40012E8: map_doit (rtld.c:609)
==149618==    by 0x401DCA9: _dl_catch_exception (dl-error-skeleton.c:208)
==149618==    by 0x401DD52: _dl_catch_error (dl-error-skeleton.c:227)
==149618==    by 0x40024DF: do_preload (rtld.c:780)
==149618==    by 0x40024DF: handle_preload_list (rtld.c:881)
==149618==    by 0x40061CE: dl_main (rtld.c:1690)
==149618==    by 0x401CC3A: _dl_sysdep_start (dl-sysdep.c:252)
==149618==    by 0x400204B: _dl_start_final (rtld.c:449)
==149618==    by 0x400204B: _dl_start (rtld.c:539)
==149618==    by 0x4001107: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.31.so)
==149618==    by 0x2: ???
==149618== 
==149618== HEAP SUMMARY:
==149618==     in use at exit: 0 bytes in 0 blocks
==149618==   total heap usage: 0 allocs, 0 frees, 0 bytes allocated
==149618== 
==149618== All heap blocks were freed -- no leaks are possible
==149618== 
==149618== For lists of detected and suppressed errors, rerun with: -s
==149618== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
