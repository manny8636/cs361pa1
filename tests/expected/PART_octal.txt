INTEGER: 16
Success!
